debian/python3-bocs.bocs.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
```

Comment:
Are there circumstances where we would want the BOCS service to start after the network?

Reply:
I don't follow. As you mentioned above we don't require that the BOCS has network access.

Reply:
Yup, I got confused here.